Firebase/Database/Core/FRepo.m

```diff
@@ -50,7 +50,7 @@
 #import "FValueEventRegistration.h"
 #import "FEmptyNode.h"
 
-#ifdef TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE
```

Comment:
It could be, yes. There's ~12 other uses of TARGET_OS_IPHONE in other files. Shall I change those while I'm at it?